tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -321,11 +343,10 @@ inline void add_cases(std::filesystem::path const& case_dir, std::string const&
     }
     // loop sym and batch
     for (bool const sym : {true, false}) {
-        std::string const output_prefix = sym ? "sym_output" : "asym_output";
-        for (std::string const& calculation_method : calculation_methods) {
-            std::string const batch_suffix = is_batch ? "_batch" : "";
+        for (auto const& calculation_method : calculation_methods) {
             // add a case if output file exists
-            std::filesystem::path const output_file = case_dir / (output_prefix + batch_suffix + ".json");
+            std::filesystem::path const output_file =
+                case_dir / (get_output_type(calculation_method, sym) + batch_suffix + ".json"s);
```

Comment:
It would create a duplicate case for sc_output right? For both sym=true and false?

Reply:
ah you're right, let me try to fix this